frontend/src/lib/lemon-ui/Popover/Popover.scss

```diff
@@ -114,7 +114,7 @@
     width: 0.5rem;
     height: 0.5rem;
     transform: rotate(45deg);
-    background: var(--bg-light);
```

Comment:
I assume this is safe 😊

Reply:
Yep, under the hood this actually calls `bg-light` in the old world but gets overridden in 3000 (dark or light)
https://github.com/PostHog/posthog/blob/master/frontend/src/styles/global.scss#L573